__tests__/reducers/welcome.test.js

```diff
@@ -1,9 +1,10 @@
 import welcomeReducer from '../../app/reducers/welcome'
 import { ACTIONS } from '../../app/constants/actions'
+jest.mock('../../utils/api.electron')
```

Comment:
I added that line as you suggested to fix the error i took screenshot of. It was in the app.test.js before, but after the changes started in here the welcome.test.js. This line fix it.